docs/rolling_back.md

```diff
@@ -0,0 +1,23 @@
+## How to Rollback (on Kubernetes, deployed w/ Hokusai)
+
+So, you're reading this doc, in which case either nothing is on fire and you're simply educating yourself (yay!), or, there's a problem and you need to rollback Force. Take a deep breath, and read on.
+
+1. Run `hokusai registry images`. You'll see output like the below.
+
+![list of images](images/hokusai_images.png "Hokusai Images")
+
+2. Locate the name of a tag you want to deploy. In this case, since we're rolling back, it'll be the previous tag of the form `production-YYYY-mm-dd..`
+
+![specific image](images/single_image.png "Last Prod Image")
+
+3. Locate the Circle CI deploy process that corresponded to the deploy you are rolling back to. You may need to scroll back in the build history until you find it. One such example is: https://circleci.com/workflow-run/9fec809a-d096-4fc0-88b7-82419ada519d
```

Comment:
Ok... I'm the one who has never done that before. All the steps are pretty clear, this finding the `process corresponding to the deploy found in step 1` is a bit not clear. How does one know that this is a corresponding deploy? The deploy does not seem to have any tags that mention `production-YYYY-mm-dd..` or I'm just missing it. I found the one just by looking at prod deploy on that day, but that might be confusing if there are several in . close time frame.

Maybe there can be an image here that shows how does one know that this deploy corresponds to the tag found above?

Reply:
Good point @oxaudo ! That is exactly how I found the corresponding Circle CI process...I scrolled back in the history on Force until I was on the day of that last deploy...and then looked at the list for a Circle CI `release` branch build.

You're totally right that can possibly be made easier, and can be confusing with multiple deploys on that day, etc.